rust/src/storage/gcs/mod.rs

```diff
@@ -41,6 +41,7 @@ impl From<tame_gcs::objects::Metadata> for ObjectMeta {
         ObjectMeta {
             path: metadata.name.unwrap(),
             modified: metadata.updated.unwrap(),
+            size: metadata.size.map(|s| s.try_into().unwrap()),
         }
```

Comment:
this should probably be fallible, but since this was not introduced here, and we are hoping to have object_store_rs soon, this is likely a topic for another day :).

Reply:
Yeah I agree. I will make a note to look at that in object_store_rs